aruco_pose:
  ros__parameters:
    cam_translation:
      x: -0.005 # camera location w.r.t to the flange
      y: 0.097 #
      z: 0.075
    cam_rotation: # These are expressed in RPY angles
      cam_alpha: 0.0
      cam_beta: 0.0
      cam_gamma: 0.0
    camera_tf_frame: 'camera_base_map'
    sample_name: 'sample_1'
    pre_pickup_location: # Define x, y, z adjustments with respect to tag's coordinate reference frame for pre-pickup
      name: 'pre_pickup'
      x_adj: 0.0
      y_adj: 0.0
      z_adj: 0.08
    offset_on_marker_x: -0.005
    offset_on_marker_y: 0.0
    fiducial_marker_family: 'DICT_APRILTAG_36h11'
    physical_marker_size: 0.02665 # measure the printed tag size
    image_topic: "/rgb/image_raw"
    number_of_observations: 10 # size of the moving window for the median filter
    cam_to_lense_x: -0.032
    cam_to_lense_y: 0.0
    cam_to_lense_z: 0.0